File: 2000/01/main.go
package main

import (
	"os"

	"aoc.com/tcollier"
)

func part1Answer(input tcollier.Input) string {
	return input.([]string)[0]
}

func part2Answer(input tcollier.Input) string {
	return input.([]string)[1]
}

func main() {
	tcollier.Executor([]string{"Hello", "World!"}, part1Answer, part2Answer, os.Args)
}

File: 2000/01/main.kt
import tcollier.AocExecutor

fun part1Answer(input: List<String>): String {
    return input[0]
}

fun part2Answer(input: List<String>): String {
    return input[1]
}

fun main(args: Array<String>) {
    val input: List<String> = listOf("Hello", "World!")
    val executor = AocExecutor(input, ::part1Answer, ::part2Answer)
    executor.run(args)
}
